proto/asn1-pdu/x509_certificate_to_der.h

```diff
@@ -51,6 +51,11 @@ DECLARE_ENCODE_FUNCTION(TBSCertificateSequence);
 DECLARE_ENCODE_FUNCTION(VersionNumber);
 DECLARE_ENCODE_FUNCTION(ValiditySequence);
 DECLARE_ENCODE_FUNCTION(TimeChoice);
+DECLARE_ENCODE_FUNCTION(Extension);
+DECLARE_ENCODE_FUNCTION(RawExtension);
+std::vector<uint8_t> GetOIDForExtension(const Extension& val);
```

Comment:
You don't need to put this in the header. It's an implementation detail of your .cc :)